notebooks/official/pipelines/custom_tabular_classification_bq_io_pipeline.ipynb

```diff
@@ -0,0 +1,1461 @@
+{
```

Comment:
<p>below code -&gt; code below</p><p>Sklearn -&gt; scikit-learn</p><p> </p><p>For more details, check <a href="https://scikit-learn.org/stable/modules/generated/sklearn.pipeline.Pipeline.html" target="_blank">Sklearn pipelines</a>. -&gt; use format:</p><p> </p><p>Learn more about [scikit-learn pipelines](link)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/992/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>